frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -444,7 +444,7 @@ const fetchNodeMetrics = (): Promise<NodeMetrics> => {
     },
     {
```

Comment:
You need to change the `usedStorage` query above to match. I believe it should be

```
sum by (instance) (node_filesystem_size_bytes{fstype!=""} - node_filesystem_avail_bytes{fstype!=""})
```